### ISSUE TITLE
`eslint-plugin-jsx-a11y` ver 3.0.1 is not working

### ISSUE THREAD

**Issue by hanagejet** [airbnb/javascript#1162]
```
	Configuration for rule "jsx-a11y/heading-has-content" is invalid:
	Value "" is the wrong type.
```

and some bug message...

issue code: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react-a11y.js#L12

Could I fix this??

**Comment by lefeverd** [comment]
Apparently it's due to eslint-plugin-jsx-a11y breaking change introduced in:
https://github.com/evcohen/eslint-plugin-jsx-a11y/commit/c8a3da94e49b89ade5e294a1936a1bb996015c25

**Comment by MrSauceman** [comment]
Found another one:

````
Configuration for rule "jsx-a11y/anchor-has-content" is invalid:  Value "" is the wrong type.

**Comment by ljharb** [comment]
It's not supposed to work - it's not a compatible peer dep with this package.

Please use the shell command to install peer deps instead of updating individual deps yourself.

**Comment by ljharb** [comment]
In other words, stick with v2.2.3

**Comment by hanagejet** [comment]
@ljharb Ok, thank you :)

**Comment by ljharb** [comment]
Just published v14.0.0 of the main config, and v11.0.1 of the base config.

(#1162, #1166, #1182, #1186, #1207, #1212, #1222)
